shell/platform/linux/fl_renderer_headless.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_LINUX_FL_RENDERER_HEADLESS_H_
+#define FLUTTER_SHELL_PLATFORM_LINUX_FL_RENDERER_HEADLESS_H_
+
+#include <gdk/gdkx.h>
```

Comment:
Is this needed?

Reply:
Nope. that's copypasta from `FlRendererX11`.